site2/docs/client-libraries-cpp.md

```diff
@@ -4,298 +4,103 @@ title: Pulsar C++ client
-| pulsar-client-debuginfo | Debug symbols for `libpulsar.so` |
-
-#### Debian
-
-To build Debian packages, enter the following command.
-
-```shell
-pulsar-client-cpp/pkg/deb/docker-build-deb.sh
-```
-
-Debian packages are created in the `pulsar-client-cpp/pkg/deb/BUILD/DEB/` path.
-
-| Package name | Content |
-|-----|-----|
-| pulsar-client | Shared library `libpulsar.so` and `libpulsarnossl.so` |
-| pulsar-client-dev | Static library `libpulsar.a`, `libpulsarwithdeps.a` and C++ and C headers |
-
-## MacOS
-
-### Compilation
-
-1. Clone the Pulsar repository.
-
-```shell
-git clone https://github.com/apache/pulsar
-```
-
-2. Install all necessary dependencies.
-
-```shell
-# OpenSSL installation
-brew install openssl
-export OPENSSL_INCLUDE_DIR=/usr/local/opt/openssl/include/
-export OPENSSL_ROOT_DIR=/usr/local/opt/openssl/
-
-# Protocol Buffers installation
-brew install protobuf boost boost-python log4cxx
-# If you are using python3, you need to install boost-python3 
-
-# Google Test installation
-git clone https://github.com/google/googletest.git
-cd googletest
-git checkout release-1.12.1
-cmake .
-make install
-```
-
-3. Compile the Pulsar client library in the repository that you cloned.
-
-```shell
-cd pulsar-client-cpp
-cmake .
-make
-```
-
-### Install `libpulsar`
-
-Pulsar releases are available in the [Homebrew](https://brew.sh/) core repository. You can install the C++ client library with the following command. The package is installed with the library and headers.
-
-```shell
-brew install libpulsar
-```
-
-## Windows (64-bit)
-
-### Compilation
-
-1. Clone the Pulsar repository.
-
-```shell
-git clone https://github.com/apache/pulsar
-```
-
-2. Install all necessary dependencies.
-
-```shell
-cd ${PULSAR_HOME}/pulsar-client-cpp
-vcpkg install --feature-flags=manifests --triplet x64-windows
+rpm -ivh apache-pulsar-client*.rpm
 ```
 
-3. Build C++ libraries.
-
-```shell
-cmake -B ./build -A x64 -DBUILD_PYTHON_WRAPPER=OFF -DBUILD_TESTS=OFF -DVCPKG_TRIPLET=x64-windows -DCMAKE_BUILD_TYPE=Release -S .
-cmake --build ./build --config Release
-```
+Now, you can see Pulsar C++ client libraries installed under the `/usr/lib` directory.
 
 :::note
 
-* For Windows 32-bit, you need to use `-A Win32` and `-DVCPKG_TRIPLET=x86-windows`.
-* For MSVC Debug mode, you need to replace `Release` with `Debug` for both `CMAKE_BUILD_TYPE` variable and `--config` option.
+If you get an error like "libpulsar.so: cannot open shared object file: No such file or directory" when starting a Pulsar client, you need to run `ldconfig` first.
 
 :::
 
-4. Client libraries are available in the following places.
+### Source
 
-```
-${PULSAR_HOME}/pulsar-client-cpp/build/lib/Release/pulsar.lib
-${PULSAR_HOME}/pulsar-client-cpp/build/lib/Release/pulsar.dll
-```
+If you want to build the Pulsar C++ client from source code, read [these instructions](https://github.com/apache/pulsar/tree/master/pulsar-client-cpp#compilation).
```

Comment:
See https://github.com/apache/pulsar/pull/17580#pullrequestreview-1105841168 and https://github.com/apache/pulsar/pull/17580#issuecomment-1245551732 for background.

@Anonymitaet could you help with reviewing this wording?